src/components/Gallery/Images.jsx

```diff
@@ -0,0 +1,3 @@
+export default function Images(source) {
+  return <img src={source} alt="" />;

```

Comment:
Now that you fixed the JSON file to only have the image name, we need to create a variable and define the correct path to the images:

since we're putting our images in the public folder then in a `gallery`(public/gallery/), we can create something like:
```js
const IMAGE_PATH = '/gallery/';
```

Remember `/` is equal `/public`.

After that we can concatenate this path with the name of the image passed from props:

```suggestion
  return <img src={PATH_IMAGE + source} alt="" />;
```